examples/aya-tool/myapp/src/main.rs

```diff
@@ -8,20 +9,21 @@ use std::{
     thread,
     time::Duration,
 };
-use structopt::StructOpt;
 
 fn main() {
     if let Err(e) = try_main() {
         eprintln!("error: {:#}", e);
     }
 }
 
-#[derive(Debug, StructOpt)]
-struct Opt {}
+#[derive(Debug, Parser)]
+struct Cli {}
 
 fn try_main() -> Result<(), anyhow::Error> {
-    // command-line options a currently unused
-    let _opt = Opt::from_args();
+    // command-line options are currently unused
+    // follow https://docs.rs/clap/4.1.4/clap/_derive/_tutorial/index.html
+    // when adding command-line argumens
+    let _cli = Cli::parse();
```

Comment:
Maybe just remove it? We are not generating the `clap` stuff anymore in aya-template unless we have some arguments to handle, right?

Reply:
sure, that's even better 😄